src/components/common/Input.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Input = ({ id, name, label, type, placeholder, value, change }) => (
+  <div className="form-group">
+    <label htmlFor={id} className="form-label">
+      {label}
+    </label>
+    <input
+      type={type}
+      className="form-control form-input"
+      id={id}
+      name={name}
+      placeholder={placeholder}
+      value={value}
+      onChange={change}
+    />
+  </div>
+);
+
+Input.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
```suggestion
  label: PropTypes.node,
```
I think this is what you are looking for, a node is anything that can be rendered to the dom. (strings, numbers, html or react elements, etc..)